sharktank_models/punet/sharded_toy_resnet_block/README.md

```diff
@@ -0,0 +1,5 @@
+### Asset generation for tests
+```
+python -m sharktank.models.punet.export_sharded_toy_resnet_block_iree_test_data
+```
```

Comment:
Thanks for the script! Can you also add a bit more context like on https://github.com/iree-org/iree-test-suites/blob/main/sharktank_models/llama3.1/README.md ? Maybe we could also merge the two into a single "generate all toy models" set of instructions

I'd specifically be looking for
1. What to install such that `sharktank.models....` works
2. Any requirements around operating system, python version, HF_TOKEN authentication, etc.